.github/workflows/test_openvino.yml

```diff
@@ -39,6 +39,15 @@ jobs:
             "*quantization*",
           ]
         transformers-version: ["4.45.0", "latest"]
+        export_mode: ["torch_script", "torch_export"]
+        include:
+          - export_mode: "torch_export"
+            use_torch_export_bool: "True"
+          - export_mode: "torch_script"
+            use_torch_export_bool: "False"
+        exclude:
+          - transformers-version: "4.45.0"
+            export_mode: "torch_export"
```

Comment:
we can't double the workload of our ci for a parameter that fails most of the time 😅